scripts/ci/check.sh

```diff
@@ -12,6 +12,6 @@ cd "$(dirname "$0")"/../..
 # Ignore possible failures there because these tests depends on measurements
 # and systems might differ in performance.
 export CI="true"
-make check
+make -j $(nproc) check
```

Comment:
This may cause quite a few hard to debug problems. Most of the validation tests and example applications are implemented with an assumption that they are the only concurrently running ODP application on the system.

Some of the validation tests are also launching more than one workers threads, which will increase the possibility of problems (e.g. time measurements issues).